.ci/lib.sh

```diff
@@ -331,15 +333,21 @@ gen_clean_arch() {
 	info "Remove installed containerd-cri related binaries and configuration"
 	delete_containerd_cri_stale_resource
 	info "Remove installed kubernetes packages and configuration"
+	[ "$(pgrep kubelet)" != "" ] && sudo sh -c 'kubeadm reset -f'
```

Comment:
I suppose we need this because it is suspected something else was broken before it. How do we know the system  behaves ok after this line?

Reply:
sorry, I'm not catch your meaning. what's your meaning by "system ok". does the command "kubeadm reset -f" do something harmful to the system?
In my case, sometimes k8s is running before ci, so we must stop it. that's why I use "kubeadm reset -f". Do you have better  method to stop k8s service?

Reply:
What am saying is that "kubeadm reset -f" is used in order to "fix" a previous issue.
Once you run this command you assume the problem is fixed.
Is there some way to ensure the problem is fixed?"

Reply:
That's a good idea to test if the "k8s" is off. But I don't know what we can if it doesn't work. What about checking if kubelet is running and give a warning if it is?

Reply:
@jongwu if you don't have an easy way to ensure the kubeadm configuration is fine now, just disregard  my comment.